pyproject.toml

```diff
@@ -1,66 +1,61 @@
 [build-system]
-requires = ["setuptools>=61.0"]
```

Comment:
When doing the release to pypi last week (master branch), @vict0rsch  made the switch from setuptools to poetry. I forgot the reason. @vict0rsch, care to remind us ?
I figured we might just as well do the switch now in generic-gfn, rather than having to deal with both whenever we switch branches.
Things work nicely with poetry however.

Reply:
OK if things work well with poetry when pip installing from `pypy` I have no qualms :)